resources/core/charts/monitoring/charts/alertmanager/README.md

```diff
@@ -1,10 +1,10 @@
 ### Alertmanager
 
-In Kyma all the configuration related to the Alertmanager is in this chart.
+In Kyma part of the configuration related to the Alertmanager is in this chart.
 
 #### Secret configuration
 
-Alertmanager instances require the secret resource naming to follow the format alertmanager-{ALERTMANAGER_NAME}.
+Alertmanager instances require a secret resource to be named with the following format alertmanager-{ALERTMANAGER_NAME}.

```

Comment:
``Alertmanager instances require a Secret resource named with the `alertmanager-{ALERTMANAGER_NAME}` format.``

Reply:
done

Reply:
I do not see any change here. Should be:
``Alertmanager instances require a Secret resource named with the `alertmanager-{ALERTMANAGER_NAME}` format.``